zetaclient/orchestrator/orchestrator.go

```diff
@@ -666,28 +667,18 @@ func (oc *Orchestrator) ScheduleCctxSolana(
 // runObserverSignerSync runs a blocking ticker that observes chain changes from zetacore
 // and optionally (de)provisions respective observers and signers.
 func (oc *Orchestrator) runObserverSignerSync(ctx context.Context) error {
-	// sync observers and signers right away to speed up zetaclient startup
-	if err := oc.syncObserverSigner(ctx); err != nil {
-		oc.logger.Error().Err(err).Msg("runObserverSignerSync: syncObserverSigner failed for initial sync")
-	}
-
-	// sync observer and signer every 10 blocks (approx. 1 minute)
-	const cadence = 10 * constant.ZetaBlockTime
-
-	ticker := time.NewTicker(cadence)
-	defer ticker.Stop()
+	// every other block
+	const cadence = 2 * constant.ZetaBlockTime
```

Comment:
Correct. And I also thought that 99% of the time nothing happens to the chain params, BUT if we alter the config, we want to reflect it in zetaclient as soon as possible. It will end up as a `noop` for most of the time, so it’s not a big deal in terms of increased ticks.

I would keep it